### DISCUSSION TITLE
Only storing the files that will be edited in the field in QFieldCloud

### DISCUSSION THREAD

**Discussion by geoclimb-andrew** [opengisch/QField#3052] [Q&A]
I am a geologist and have been using QField with our geological team for a few years now and am currently trying to integrate our work flow into QFieldCloud.
Often our work is in remote areas and we have to rely on satellite internet, which is problematic when uploading or downloading large files.
The QField project that I am currently working with is 6.05 Giggabytes in size and includes many large layers that are only being used for reference that we commonly turn off and on when working in the field.
The “data.gpkg” file is only 292 kilobytes and the qgs and qgs~ files are only 4 megabytes.
I would like to add the reference files directly to the tablets using a usb cable and only store the files that will be edited in the field and in need of syncing in QFieldCloud.
So far I have not had much success. It seems QFieldCloud wants all the files in the cloud and ends up aborting the upload as the internet is not good enough.
Is there a suggested work flow to make this happen or am I out of luck and have to revert back to the old system of my team handing their tablets back to me for syncing on to the one central computer.
Thanks

**Comment by lucienicolier** [comment]
Hi @geoclimb-andrew,

It has never been tested. It might work, but I don't know the outcome, except the warning messages.

Have a try following this [outside layers](https://docs.qfield.org/fr/how-to/outside-layers/)

I hope it will help, let me know.
